docker-compose.yml

```diff
@@ -56,9 +55,7 @@ services:
       - "-Dkeycloak.migration.file=/data/renga-realm.json"
       - "-Dkeycloak.migration.strategy=IGNORE_EXISTING"
     volumes:
-      - ./services/keycloak/docker-entrypoint.sh:/data/docker-entrypoint.sh:ro
-      - ./services/keycloak/renga-realm.json.tpl:/data/renga-realm.json.tpl:ro
-      - ./services/keycloak/data:/data:rw
+      - ./services/keycloak/renga-realm.json:/data/renga-realm.json
```

Comment:
Not read-only?

Reply:
actually I couldn't see any difference on the resulting file permissions when I used the flags in docker-compose

Reply:
This does not necessarily translates into file permissions, but the volume driver will effectively forbid writting of the file